dojo/models.py

```diff
@@ -1389,15 +1388,17 @@ def get_breadcrumbs(self):
                 'url': reverse('view_test', args=(self.id,))}]
         return bc
 
-    def verified_finding_count(self):
-        return self.finding_set.filter(verified=True).count()
+    # def verified_finding_count(self):
```

Comment:
can be deleted if it is not used anymore?

Reply:
done